src/app/localStorage.tsx

```diff
@@ -0,0 +1,16 @@
+class StoreData {
+    data: {[key: string]: any}
+    constructor() {
+        this.data = this.loadData()
+    }
+
+    saveInfo = (data: {[key: string]: any}) => localStorage.setItem('userInformation', JSON.stringify(data))
+
+    loadData = () => localStorage.getItem('userInfo') || {}
+
+    getData = (key: string) => this.data[key]
+}
+
+
+const storage = new StoreData()
+export default storage;
```

Comment:
Why are using local storage?

Reply:
To store abit of data about logged in user, eg role and later we can retrieve and use to perform other operations.